src/Components/AddNewProject/AddNewProject.tsx

```diff
@@ -0,0 +1,111 @@
+
+import axios from "axios";
+import { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { AuthContext } from "../../Context/AuthContext";
+
+
+
+export default function AddNewProject() {
+
+
+  const navigate = useNavigate();
+  const {baseUrl , requstHeaders} : any = useContext(AuthContext);
+  const [isLoding , setIsLoding] = useState(false);
+
+
+  const {
+      register,
+      handleSubmit,
+      formState : { errors },
+  } = useForm();
+
+  // ****************** to add new project **********************
+  const onSubmit = (data)=> {
+      // console.log(data);
+      setIsLoding(true);
+
+      axios.post(`${baseUrl}/Project` , data ,
+      {
+          headers : requstHeaders,
+      })
+      .then((response)=> {
+          console.log(response);
```

Comment:
Here i just want you to know that we don't leave the console.log like that, in the branch that we use for deployment we don't leave this console.logs